### ISSUE TITLE
menu, filter-menu: aria-labelledby and keydown event for Arrow keys

### ISSUE THREAD

**Issue by cynthiahhc** [eBay/ebayui-core#2240]
# Bug Report

## Description

For both `ebay-menu` and `ebay-filter-menu`:

1. We cannot pass the `aria-labelledby` to the correct element. It can only be added to the root element, not the element with the `role="menu"`.
2. Need to emit the `keydown` event for both `ArrowUp` and `ArrowDown`. Currently, it only emits keydown events for `ESCPAE` and action keys.

One of the issue has been filed before but never resolved. #1916 

## Workaround

No
